Tests/MongoSwiftSyncTests/UnifiedTestRunner/EntityDescription.swift

```diff
@@ -1,5 +1,7 @@
 import Foundation
+@testable import struct MongoSwift.MongoClientOptions
```

Comment:
I was going to rename this file to just Entity.swift since it contains various Entity-related code now but GitHub considered it too big a diff and was treating it as a new file. so I will do that in a future PR